openshift/templates/cachito-api.yaml

```diff
@@ -0,0 +1,140 @@
+kind: Template
+apiVersion: v1
+metadata:
+  name: cachito-api
+  description: >
+    cachito-api
+
+labels:
+  build: cachito-api
```

Comment:
nitpick: `app` instead of `build` (otherwise it shows up under `Other resources` in OpenShift console)